tests/hydrothermal_cooling.prm

```diff
@@ -0,0 +1,251 @@
+    # of the first time step when the velocity has not been determined yet. 
+    set Minimum strain rate = 1.e-20
+    set Reference strain rate = 1.e-16
+
+    # Limit the viscosity with minimum and maximum values
+    set Minimum viscosity = 1e19
+    set Maximum viscosity = 1e25
+
+    # Thermal diffusivity is adjusted to match thermal conductivities
+    # assumed in assigning the initial geotherm
+    set Define thermal conductivities = true
+    set Thermal conductivities = 3.0, 1.5, 3.0
+    #set Thermal diffusivities = 1.333333e-6, 1.190476e-6,  1.333333e-6
+    set Heat capacities       =        750.,        750.,   750.
+    set Densities             =        3300,        2800,   3300
+    set Thermal expansivities =        2e-5,        2e-5,   2e-5
+
+    # Harmonic viscosity averaging
+    set Viscosity averaging scheme = harmonic
+
+    # Choose to have the viscosity (pre-yield) follow a dislocation
+    # diffusion or composite flow law.  Here, dislocation is selected
+    # so no need to specify diffusion creep parameters below, which are
+    # only used if "diffusion" or "composite" option is selected.
+    set Viscous flow law = dislocation
+
+    # Dislocation creep parameters for 
+    # 1. Background material/mantle (dry olivine)
+    #    Hirth & Kohlstedt (2004),  Geophys. Monogr. Am. Geophys. Soc., v.138, p.83-105.
+    #    "Rheology of the upper mantle and the mantle wedge:a view from the experimentalists"
+    # 2. Upper crust (wet quartzite)
+    #    Rutter & Brodie (2004), J. Struct. Geol., v.26, p.2011-2023.
+    #    "Experimental grain size-sensitive flow of hot-pressed Brazilian quartz aggregates"
+    # 3. Lower crust and weak seed (wet anorthite)
+    #    Rybacki et al. (2006), J. Geophys. Res., v.111(B3).
+    #    "Influence of water fugacity and activation volume on the flow properties of fine-grained    
+    #    anorthite aggregates"
+    # Note that the viscous pre-factors below are scaled to plane strain from unixial strain experiments.
+    set Prefactors for dislocation creep          = 6.52e-16, 8.57e-28, 6.52e-16
+# 7.13e-18
+    set Stress exponents for dislocation creep    =      3.5,      4.0, 3.5
+#,    3.0
+    set Activation energies for dislocation creep =   530.e3,   223.e3, 530.e3
+#,  345.e3
+    set Activation volumes for dislocation creep  =   18.e-6,       0., 18.e-6
+
+    # Plasticity parameters
+    set Angles of internal friction =   20.,   20., 20.
+    set Cohesions                   = 20.e6, 20.e6, 5.e6
+
+  end
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 9.81
+  end
+end
+
+
+# Post processing
+subsection Postprocess
+  set List of postprocessors = velocity statistics, temperature statistics, heating statistics, visualization
+  subsection Visualization
+    set List of output variables = density, viscosity, strain rate, thermal conductivity
+    set Output format = vtu
```

Comment:
```suggestion
    set Output format = gnuplot
```

The gnuplot format is used for tests as it can be easily compared (with `diff`) by the tester.

Reply:
Please also put the gnuplot results file in the tests/hydrothermal_cooling folder, so that the thermal conductivity is checked.